app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -497,22 +498,28 @@ fun ParkingDetailsScreen(
                                   text = stringResource(R.string.card_screen_user),
                                   style = MaterialTheme.typography.bodyMedium,
                                   color = MaterialTheme.colorScheme.onBackground)
-                              Text(
-                                  text =
-                                      stringResource(
-                                          R.string.display_user_tag_format,
-                                          range.symbol,
-                                          level,
-                                          ownerUsername),
-                                  style = MaterialTheme.typography.bodyMedium,
-                                  color =
-                                      if (range.color ==
-                                          stringResource(R.string.rainbow_text_color)) {
-                                        MaterialTheme.colorScheme
-                                            .onSurface // TODO rainbow text color
-                                      } else {
-                                        Color(parseColor(range.color))
-                                      })
+                              if (range.color == stringResource(R.string.rainbow_text_color)) {
```

Comment:
kind of a shame that the Text composable is duplicated once again, but I get why so it's non blocking for me

Reply:
Glad that you understand my point of view.

Reply:
I don't...